guides/templates_ecr.md

```diff
@@ -0,0 +1,190 @@
+
+Replace the export function with the below code. _If desired, change the EcrRepo name from "Frontend"._
+
+    export class AppCosmosStack extends CosmosExtensionStack {
+      readonly codeRepo: CodeRepository;
+      readonly ecrRepo: EcrRepository;
+
+      constructor(scope: Construct, id: string, props?: StackProps) {
+        super(scope, id, props);
+
+        this.codeRepo = new CodeRepository(this, "CodeRepo", {
+          repositoryName: this.nodeId("Code-Repo", "-").toLowerCase(),
+        });
+
+        this.ecrRepo = new EcrRepository(this, "EcrRepo", {
+          repositoryName: this.nodeId("Frontend", "/").toLowerCase(),
+        });
+      }
+    }
+
+### bin/galaxy.ts
+This file will remain unchanged, as we don't need to create any resources at the galaxy (account) level. Resources are either scoped to the cosmos (management account) or solar system (dev, test, prod environment etc) level.
+
+### bin/solar-system.ts
+
+You will need to import the following constructs:
+
+    import { Repository } from '@aws-cdk/aws-ecr';
+    import { ContainerImage } from '@aws-cdk/aws-ecs';
+    import { SsmState } from '@cosmos-building-blocks/common';
+    import { EcsService } from '@cosmos-building-blocks/service';
+
+Add app version to solar system props
+
+    export interface AppSolarSystemProps extends SolarSystemExtensionStackProps {
+      appVersion?: string;
+    }
+
+Add the following inside the constructor, after `super();`
+
+    const { appVersion } = props || {};
+    const { ecrRepo } = this.galaxy.cosmos;
+    const { vpc } = this.portal;
+    const { cluster, httpListener, httpsListener } = this.portal.addEcs();
+    const ecrRepoClone = Repository.fromRepositoryAttributes(this, 'EcrRepo', ecrRepo); // Scope issue
+    const versionState = new SsmState(this, 'VersionState', {
+      name: '/' + this.nodeId('VersionState', '/'),
+      value: appVersion,
+    });
+
+    new EcsService(this, 'Frontend', {
+      vpc,
+      cluster,
+      httpListener,
+      httpsListener,
+      containerProps: {
+        image: ContainerImage.fromEcrRepository(ecrRepoClone, versionState.value),
+        port: {
+          containerPort: 3000,
+        },
+      },
+      routingProps: {
+        pathPattern: '/demo',
+        httpsRedirect: true,
+      },
+    });
+
+### bin/main.ts
+As mentioned in the intro, this file instantiates all the resources you have defined in the above files and is where you can design your Cosmos infrastructure, including AWS accounts, stacks and pipeline stages.
+
+Add the following to imports
+
+    import { AccountPrincipal } from '@aws-cdk/aws-iam';
+
+In this file, you set the environment configuration for each galaxy and then instantiate a galaxy per environment. Examples are included in the blank template. Duplicate these and replace the names as needed. It is essential that names match those set in your core CDK-Cosmos, e.g. `Dev` must be consistently named.
```

Comment:
Galaxy per account rather than environment. "dev" and "tst" are two environments that would live in the one "dev" account